### PR TITLE
Add more examples to readme file

### PR DESCRIPTION
**Pull request by dusan-kovacik-profiq**
# Description

# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

### CODE REVIEW

**Review by lorigoldman — APPROVED** [review]

**Comment by lorigoldman** [comment]
In [login_and_registration_flow.png], please initial cap the "R" in register.

**Comment by lorigoldman** [comment]
In [oidc_with_continuation.png], please initial cap the "F" in finish.

**Comment by lorigoldman** [comment]
In reset_password_with_switch, please initial cap the "F" in forgotten, and the "R" in register.

File: README.md [hunk 1]
```diff
@@ -211,37 +386,36 @@ const IKForm = () => {
...
+The `register` branch will be used if the condition specified inside the Switch node is fulfilled. By default there's a following condition:
 
-Use the `onRenderComponent` property to tweak an existing element or create a new one. In this example we will put a small _"Enter the same password here"_ message under the second password input in the registration form.
+```
+has(session.input) && has(session.input.flow) && session.input.flow=='register'
+```
 
-```javascript
+It checks whether the input object has a property `flow` set to `register`. You can define the input object with passing the `arguments` property to your `IKUICore.renderForm` call:
+
```
Comment (hanny21) [inline README.md:396]: as an external developer I would probably wonder if there is different case than `register` :thinking:

**Review by hanny21 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -328,53 +721,354 @@ const IKForm = () => {
...
-  return <span>OIDC login...</span>;
+  return (
+    <div>
+      <div id="form-container" style={{ width: 350 }} />
+    </div>
+  );
+};
+```
+
+### How to add action button translations
```
Comment (lorigoldman) [inline README.md:992]: Suggestion: Add action button translations

File: README.md [hunk 3]
```diff
@@ -328,53 +721,354 @@ const IKForm = () => {
...
+
+In the authentication builder in action nodes you can specify your own locale keys. Then using the `actionLabels` property you can map your custom messages.
```
Comment (lorigoldman) [inline README.md:994]: Suggestion: In the authentication builder in action nodes, ...keys. Then, using the `actionLabels` property,

**Review by lorigoldman — APPROVED** [review]
Stellar work, as usual, Dusan!

File: README.md (hunk 1 above)
Comment (dusan-kovacik-profiq) [reply README.md:396]: It's difficult to predict what a developer may need, but [here's another example](https://github.com/indykite/jarvis-sdk-web/tree/13bc5c018323e093ec8a6185e0bbb30f86f33cd5#using-a-special-url) how to use the switch node ;)

**Pull request merged**